### DISCUSSION TITLE
Clarification regarding gnomAD AF values

### DISCUSSION THREAD

**Discussion by anjalijain22** [broadinstitute/seqr#3314] [Q&A]
Hi!

I have a couple of questions regarding the gnomAD allele frequency values that Seqr shows on the variant search page:
Using this variant as an example: GNAT1 (3-50230821-C-A)

![Screen Shot 2023-04-06 at 2 06 14 PM](https://user-images.githubusercontent.com/60684431/230462630-29f229b7-d4c0-45e6-a595-ef7119dfebd4.png)

- What is the distinction between gnomAD v2 exomes and genomAD v3 genomes? Both of the hyperlinks point to the same link - https://gnomad.broadinstitute.org/gene/ENSG00000114349 which is gnomAD v2.1.1
- How are the values for "gnomAD v2 exomes **0.000018**" and "gnomad v3 genomes **0.000115**" derived? When I look at the original [gnomAD record for the variant ](https://gnomad.broadinstitute.org/variant/3-50230821-C-A), I don't see these values anywhere. When I hover over the AF in seqr, a pop-up box shows up which has the global AFs and the allele counts that correctly match up to the gnomAD variant record. Ideally, I should be seeing the same AF in the variant details as well. Same is the case for ExAC and TopMed.

Thanks in advance for your time and help with this, appreciate it!

**Comment by hanars** [comment]
So gnomad v2 is aligned to build GRCh37 while v3 is on build GRCh38. For seqr projects on GRCh38 we link to v2 and v3 correctly as we have liftunder coordinates to use, however for GRCh37 we can't link to v3 since we don't have lifted coordinates so we just link to v2 instead. The frequency shown, however, is from the v3 genomes liftunder data, so it is still accurate for the v3 data. There are also 2 links per gnomad frequency - the link for "gnomad vX" links to the gene while the actual frequency itself links to the specific 

The AF shown on the variant is the AF seqr uses for filtering, which is the popmax AF. Gnomad currently does not display the true popmax AF in the browser, they display Popmax Filtering AF, but we find for rare disease analysis that the true popmax is better
